.eslintrc

```diff
@@ -15,7 +15,7 @@
   },
   "overrides": [
     {
-      "files": [ "stories.js", "test.js", "src/setupTests.js", "scripts/createComponent.js", "src/stories/**" ],
+      "files": [ "stories.js", "test.js", "src/setupTests.js", "scripts/createComponent.js", "src/stories/**", "src/hoc/*/stories.js", "**/stories.js" ],
```

Comment:
I don't understand why `"src/hoc/*/stories.js"` is needed if we already have "**/stories.js"`

Reply:
similarly, `stories.js` should pick up all files with this name, doesn't it?